deploy/base/kustomization.yaml

```diff
@@ -2,8 +2,9 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
+  - gradle-v0.1.yaml
   - https://github.com/tektoncd/pipeline/releases/download/v0.34.1/release.yaml
-  - https://raw.githubusercontent.com/redhat-appstudio/build-definitions/main/tasks/git-clone.yaml
+  - https://raw.githubusercontent.com/tektoncd/catalog/main/task/git-clone/0.5/git-clone.yaml
```

Comment:
This looks like a rebase issue.

Reply:
I reverted that change to make the script still using the old task work. I think need to change the parameters to the new ones.

Reply:
Reverted back to upstream. It works now.